scripts/find_unused_deps.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'toml-rb'
+require 'set'
+
+exit_code = 0
+
+Dir.glob("#{File.dirname(File.expand_path($PROGRAM_NAME))}/../**/*.toml").each do |file|
+  crate_dir = File.dirname(file)
+  toml = TomlRB.load_file(file)
+  crates = Set.new
+  toml['dependencies']&.each do |pair|
+    crates.add pair[0]
+  end
+  toml['dev-dependencies']&.each do |pair|
+    crates.add pair[0]
+  end
```

Comment:
This should work, but please double-check. I think it's more expressive.
```suggestion
  toml['dependencies']&.each do |crate_name, _|
    crates.add crate_name
  end
  toml['dev-dependencies']&.each do |crate_name, _|
    crates.add crate_name
  end
```

Reply:
Fixed, it works